src/main/java/io/xpring/payid/javascript/JavaScriptPayIDUtils.java

```diff
@@ -41,8 +41,8 @@ public JavaScriptPayIDUtils() throws JavaScriptLoaderException {
   public PayIDComponents parsePayID(String payID) {
     Objects.requireNonNull(payID);
 
-    Value parsePayIDFunction = javaScriptPayIDUtils.getMember("parsePaymentPointer");
-    Value javaScriptComponents = parsePayIDFunction.execute(payID);
+    Value parsePayIDFunction = javaScriptPayIDUtils.getMember("parsePayID");
+    Value javaScriptComponents =  parsePayIDFunction.execute(payID);
```

Comment:
nit: extra space (Does Java have a linter? or Java-Prettier?)

Reply:
Fixed. #193 enabled the linter under pretty strict rules so you would think that it would complain. 

I don't feel like I have time at the moment to really dig into this; but there is lint support for the project and I did fix the spacing.

Reply:
No worries, just seemed like the sort of nit that could potentially be automated.